### ISSUE TITLE
Inadequate contrast and screen reader compatibility: sort icons on admin browse views

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1053]
On admin browse pages, sort icons have inadequate contrast (1.2:1) against table headers:
<img width="753" alt="Screenshot detail of sort icons on table header. A Colour Contrast Analyzer window is open, indicating a 1.2:1 contrast" src="https://github.com/user-attachments/assets/3e1963bc-c4e5-4ef6-8ba3-1ad4444715a4" />
These elements should be at least 3:1 with the background.

**Comment by kimisgold** [comment]
I've just pushed a fix. It should bump up the contrast of the default inactive sorting icons. Active sorting icons should be full opacity. As a note, the active icons are single arrows pointing up or down that are layered over very light transparent up-down arrows to create the look of one direction being inactive. This is a visual style preference, as the single directional arrows looked imbalanced to me. I don't think we should have to worry about the contrast of the background set of arrows. This is what an active state should look like.

![Screenshot of "Date Added" column heading with active ascending arrow](https://github.com/user-attachments/assets/712ebadc-e638-4b3a-8cb1-6f43b8d2d073)

This is what the default column heading state should look like, with the two-directional arrows at 75% opacity.

![Screenshot of "Title" column heading with default two-directional arrows](https://github.com/user-attachments/assets/2c1c2400-4e80-4d08-88f5-afe5367ea71e)

**Comment by aviannamiller** [comment]
I'm seeing the new style in the code but not on the interface; it does not appear to be pulling over to these kinds of responsive tables.

**Comment by aviannamiller** [comment]
My apologies, I am seeing it now! I am getting some separation of the arrow overlay when header text overflows beyond one line:
<img width="92" alt="Screenshot of 'Date Added' column with active ascending arrow floating over two inactive ascending/descending arrows" src="https://github.com/user-attachments/assets/7c725940-333e-4645-824d-d4ee0932bfdb" />

**Comment by aviannamiller** [comment]
In terms of screen reader behavior, the page reload that is prompted when a sort mode is chosen causes lost focus. Is there a way to either return focus immediately to the trigger element on reload or to make the reload dynamic so that the page does not have to reload in entirety? Perhaps part and parcel to this page reload, there is no screen reader announcement of the state change when the sort icon is activated to change sort direction. Also, matching `aria-label` and `title` attributes on column headers are causing a duplicate readout on all tested screen readers.

**Comment by kimisgold** [comment]
Good catch with the responsive positioning, I can definitely address that, along with the duplicate information in the `aria-label` and `title` attributes.

> Is there a way to either return focus immediately to the trigger element on reload or to make the reload dynamic so that the page does not have to reload in entirety?

This would be a pretty massive undertaking. Handling it dynamically/client-side would make things much easier for screen readers, but the scope of redesigning and implementing that kind of solution across all our browse patterns is currently unfeasible. The best I can think of is sending the sort information to the page title so that it could be read out on reload. I think that's a conversation we can continue in another issue: I started [this one](https://github.com/omeka/Omeka/issues/985) when it came up in a previous assessment.

**Comment by kimisgold** [comment]
Okay, arrow positioning and removal of the redundant aria label are ready to look at.

**Comment by aviannamiller** [comment]
Thanks, the changes look good, although I am still seeing a slight arrow misalignment at page zooms 110%+. It's very slight, but it is there and appears on all tested browsers (Firefox, Chrome, and Safari):
<img width="351" alt="Close up detail on light grey ascending and descending sort arrows. There is a slightly misaligned black descending arrow layered over the bottom arrow" src="https://github.com/user-attachments/assets/abfefcf3-119d-4272-8a02-1540e9164bbc" />

**Comment by kimisgold** [comment]
I've pushed a half pixel adjustment that should hopefully address this.

**Comment by aviannamiller** [comment]
It's fixed on desktop, but I am still seeing that slight misalignment on mobile:

![Image alt="Close up detail on light grey ascending and descending sort arrows next to the word Title. There is a slightly misaligned black descending arrow layered over the bottom arrow"](https://github.com/user-attachments/assets/d0e1b3fe-a591-48ae-8a2e-003ab2f9a66c)
![Image alt="Same detail as above, though now the misaligned black arrow is layered over the top arrow"](https://github.com/user-attachments/assets/eb71c3d6-a065-4653-8f30-79a72dcb6c35)

**Comment by kimisgold** [comment]
Okay, one more swing at this. The icon glyphs should be the same height and position layered over each other. Hopefully mobile browsers agree with how the desktops render this.

**Comment by aviannamiller** [comment]
It's perfect now, same alignment on both desktop and mobile browsers. Thank you for all the tweaks on this!
